faq.md

```diff
@@ -52,13 +52,7 @@ presentation and semantics.
 
 **How is the MathML association different from the MathJax Consortium?**
 
-As indicated on the MathJax website, "The core of the MathJax project is the
-development of its state-of-the-art, open source, JavaScript platform for
-display of mathematics.". Instead, the MathML association does not focus
-on promoting a particular product but intends to implement the MathML standard
-into Web browsers and assistive technologies. Note that MathJax's internal
-representation is MathML and its team members have advocated for native MathML
-support in Web browsers, but they have never made any official statement
-regarding funding plans. Moreover, the way MathJax encapsulates MathML makes it
-badly integrated into the Web platform and hardly accessible to MathML-aware
-tools ; hence this can only be a transition technology.
+We are happy that MathJax has already embraced MathML, which is used as its core formula representation language. While MathJax has provided an invaluable bridge solution for displaying beautiful math-on-the-web today, the MathML association is focusing on advancing the state of web mathematics tomorrow.
+
+Our goal towards standards-compliant web mathematics, via native MathML support in all browsers, aims at faster rendering, better accessibility and towards building a healthy ecosystem of math web application development. As such, our goals are potentially complementary to the MathJax Consortium and we are open to partnering with them to this end.
+
```

Comment:
> As such, our goals are potentially complementary to the MathJax we are open to partnering with them to this end.

The term "Partner" is not really in accordance with the affiliates text, since they do not do Web engine development. Also, I'm not sure one can say that the MathJax Consortium has a clear position regarding native MathML support. So I would instead suggest the following:

Although "The core of the MathJax project is the development of its state-of-the-art, open source, JavaScript platform for display of mathematics", some members of the MathJax team has publicly expressed support for native MathML in the past. Hence we hope that the MathJax Consortium will support the MathML manifesto and potentially become a MathML sponsor.'